client/driver/qemu.go

```diff
@@ -29,9 +32,15 @@ var (
 )
 
 const (
-	// The key populated in Node Attributes to indicate presence of the Qemu
-	// driver
-	qemuDriverAttr = "driver.qemu"
+	// The key populated in Node Attributes to indicate presence of the Qemu driver
+	qemuDriverAttr                = "driver.qemu"
+	qemuDriverVersionAttr         = "driver.qemu.version"
+	qemuDriverLongMonitorPathAttr = "driver.qemu.longsocketpaths"
```

Comment:
I am not a fan of this being an attribute. I think we can just check in the start method

Reply:
For scheduling purposes, constraining on this can be replicated by: https://www.nomadproject.io/docs/job-specification/constraint.html#quot-version-quot-

Reply:
Thanks - ``qemuDriverLongMonitorPathAttr`` is no longer an attribute!